### ISSUE TITLE
Use "name=..." syntax in AddDbContext

### ISSUE THREAD

**Issue by ajcvickers** [dotnet/templating#869]
Now that https://github.com/aspnet/EntityFramework/pull/8641 is merged, any templates that call AddDbContext could be updated so that they do not reference IConfiguration directly. For example:
```C#
.AddDbContext<MyContect>(
   b => b.UseSqlServer("name=DefaultConnection"))
```

@glennc @DamianEdwards @divega 
Do you guys want me to send a PR for this?

**Comment by divega** [comment]
@ajcvickers I am fine with this, although I am a bit confused by the state of the default templates at the moment :smile_cat: 

I guess the change would go in https://github.com/dotnet/templating/blob/master/template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/IdentityServiceStartup.cs#L30

**Comment by ajcvickers** [comment]
@glennc @DamianEdwards Ping to triage this. Do you guys want this in preview2? Post preview2? Not at all?

**Comment by DamianEdwards** [comment]
Would be great to have this and it seems like a very simple template change.

**Comment by ajcvickers** [comment]
Yep, very simple. I'll send a PR today.

**Comment by mlorbetske** [comment]
Fixed in #879
